static/js/apps/eval_retrieval_generation/overall_feedback.tsx

```diff
@@ -45,80 +46,118 @@ const LOADING_CONTAINER_ID = "form-container";
 // object to hold information about a question
 interface QuestionConfig {
   title: string;
-  question: string;
-  responseOptions: Record<string, string>;
+  questions: {
+    firestoreKey: string;
+    question: string;
+    responseOptions: Record<string, string>;
+  }[];
 }
 
 // Dictionary of feedback stage -> eval type -> question config
 const QUESTION_CONFIG: Record<string, Record<string, QuestionConfig>> = {
   [FeedbackStage.OVERALL_QUESTIONS]: {
     [EvalType.RAG]: {
+      questions: [
+        {
+          firestoreKey: QUERY_OVERALL_QUESTIONS_KEY,
+          question:
+            "Do the generated questions seem sufficient to answer the query?",
+          responseOptions: {
+            [OVERALL_QUESTIONS_OPTION_HAS_MISSING]: "Missing obvious questions",
```

Comment:
why the option needs a key and value constant?

Reply:
the key is the option we had set in google sheets & value is the way we display it to users